test/powershell/Language/Parser/LineContinuance.Tests.ps1

```diff
@@ -0,0 +1,247 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe 'Line Continuance' -Tags 'CI' {
+    BeforeAll {
+        function ExecuteCommand {
+            param ([string]$command)
+            [powershell]::Create().AddScript($command).Invoke()
+        }
```

Comment:
Do we need cleanup?
```powershell
        function ExecuteCommand {
            param ([string]$command)
            try {
                $PowerShell.AddScript($command).Invoke()
            }
            finally {
                $PowerShell.Commands.Clear()
            }
        }
```

Reply:
Maybe. I wasn't sure. I saw that function, but wasn't sure which is the current/best way to write these pester tests, and didn't want to take a dependency that I didn't need. FWIW, I disagree with the logic used to write the other function -- a new PowerShell instance should be created for every invocation rather than clearing the command stack between invocations of commands (I have found in practice while working with PowerShell from C# a lot that this can lead to trouble and should be avoided, especially when validating tests). If we want this to be cleaned up, I would prefer changing the other tests to use my function than changing my tests to use the other function.